README.md

```diff
@@ -1,64 +1,104 @@
+| [Eva Biesot](https://github.com/eeeevieb) | Particolarte |
+| [Karel Zijp](https://github.com/zpks) | Particolarte |
+| [Peter Sabel (Zabel)](https://github.com/A-Daneel) | Particolarte |
+| Arnold van Bemmelen | ViaKunst |
+| [David Koymans](https://github.com/DavidckPixel) | ViaKunst |
+| [Machiel Kruger](https://github.com/mkrugr) | ViaKunst |
 
-| **Name**                                           | **Organisation** |
-|----------------------------------------------------|------------------|
-| [Jasper Weyne](https://github.com/jasperweyne/)    | Project Owner    |
-| [Machiel Kruger](https://github.com/mkrugr)        | ViaKunst         |
-| [David Koymans](https://github.com/DavidckPixel)   | ViaKunst         |
-| Arnold van Bemmelen                                | ViaKunst         |
-| [Peter Sabel (Zabel)](https://github.com/A-Daneel) | Particolarte     |
-| [Borre Pol Meeuwisse](https://github.com/Borrepol) | Particolarte     |
-| Karel Zijp                                         | Particolarte     |
-| [Eva Biesot](https://github.com/eeeevieb)          | Particolarte     |
+## Contact
+You can either join the [discord](https://discord.gg/4HUmvEnXn8), file a [bug
+report][bug-create-url], or make a [feature request][feature-create-url].
 
-## How to install/deploy
-Please refer to the [INSTALL.md](INSTALL.md) file. Making a quick change? You
-can use Gitpod to quickly deploy a development environment as well!
+## License
+This work is [licensed](https://github.com/jasperweyne/helpless-kiwi/blob/develop/LICENSE) under the [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0).
 
-[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/jasperweyne/helpless-kiwi)
+[sc-gate-shield]: https://sonarcloud.io/api/project_badges/measure?project=jasperweyne_helpless-kiwi&metric=alert_status
+[sc-bugs-shield]: https://sonarcloud.io/api/project_badges/measure?project=jasperweyne_helpless-kiwi&metric=bugs
+[sc-smells-shield]: https://sonarcloud.io/api/project_badges/measure?project=jasperweyne_helpless-kiwi&metric=code_smells
+[sc-vuln-shield]: https://sonarcloud.io/api/project_badges/measure?project=jasperweyne_helpless-kiwi&metric=vulnerabilities
+[sc-duplication-shield]: https://sonarcloud.io/api/project_badges/measure?project=jasperweyne_helpless-kiwi&metric=duplicated_lines_density
+[sc-project-url]: https://sonarcloud.io/dashboard?id=jasperweyne_helpless-kiwi
+[discord-url]: https://discord.gg/4HUmvEnXn8
 
-## Commands to remember!
+[bug-create-url]: https://github.com/jasperweyne/helpless-kiwi/issues/new?assignees=&labels=&template=bug_report.md&title=
+[feature-create-url]: https://github.com/jasperweyne/helpless-kiwi/issues/new?assignees=&labels=&template=feature_request.md&title=
 
- - ```php bin/console cache:clear```: Always a good idea!
- - ```composer fix```: Fix PHP source code formatting
- - ```composer test```: Run unit tests in /tests/
```

Comment:
I agree that this might not be the best place for it, but right now, it's dropped completely. I feel that some documentation on commands should be integral to a system like this. I'd like to suggest the addition of a /docs folder, where the non-typical docs could live (DEV.md, DEPLOY.md are notable examples of documentation files that are highly useful but not usually found in the root folder, in contrast with a README.md or LICENSE file). There, documentation on typically used commands can be added.

Reply:
love this idea, will implement it in a bit.